docs/release-notes.md

```diff
@@ -1,13 +1,13 @@
 <sup><sup>*Release notes were automatically generated by [Shipkit](http://shipkit.org/)*</sup></sup>
 
 #### 2.1.4
- - 2019-03-01 - [6 commits](https://github.com/mockito/shipkit/compare/v2.1.3...v2.1.4) by [Erhard Pointl](https://github.com/epeee) (4), [shipkit.org automated bot](https://github.com/shipkit-org) (2) - published to [![Gradle](https://img.shields.io/maven-metadata/v/https/plugins.gradle.org/m2/org/shipkit/java/maven-metadata.xml.svg?colorB=007ec6&label=Gradle)](https://plugins.gradle.org/plugin/org.shipkit.java/2.1.4)
+ - 2019-03-01 - [6 commits](https://github.com/mockito/shipkit/compare/v2.1.3...v2.1.4) by [Erhard Pointl](https://github.com/epeee) (4), [shipkit.org automated bot](https://github.com/shipkit-org) (2) - published to [![Gradle](https://img.shields.io/badge/Gradle-v2.1.4-blue.svg)](https://plugins.gradle.org/plugin/org.shipkit.java/2.1.4)
  - Make sure that gitPush is executed before updateReleaseNotesOnGithub [(#782)](https://github.com/mockito/shipkit/pull/782)
  - Update objenesis (v3.0.1) and get rid of not needed dependency [(#780)](https://github.com/mockito/shipkit/pull/780)
  - Version of shipkit upgraded to 2.1.3 [(#779)](https://github.com/mockito/shipkit/pull/779)
 
 #### 2.1.3
- - 2019-02-23 - [2 commits](https://github.com/mockito/shipkit/compare/v2.1.2...v2.1.3) by [Marcin Stachniuk](https://github.com/mstachniuk) - published to [![Gradle](https://img.shields.io/maven-metadata/v/https/plugins.gradle.org/m2/org/shipkit/java/maven-metadata.xml.svg?colorB=007ec6&label=Gradle)](https://plugins.gradle.org/plugin/org.shipkit.java/2.1.3)
+ - 2019-02-23 - [2 commits](https://github.com/mockito/shipkit/compare/v2.1.2...v2.1.3) by [Marcin Stachniuk](https://github.com/mstachniuk) - published to [![Gradle](https://img.shields.io/badge/Gradle-v2.1.3-blue.svg)](https://plugins.gradle.org/plugin/org.shipkit.java/2.1.3)
```

Comment:
Minor. It would be nice to fix all badges for all versions. Btw. 2.1.4, 2.1.2, 2.1.1 didn't exist - error during deployment.

Reply:
thx, yes, I will do so.